.github/workflows/main.yml

```diff
@@ -112,7 +112,8 @@ jobs:
     - uses: actions/setup-node@v2
       with:
         node-version: '16'
-    - run: cargo test
+    # Run everything except the example tests; we run them later in `test_examples`.
+    - run: RUSTFLAGS="--cfg=disable_example_tests" cargo test
```

Comment:
Given how heavyweight these tests are in system dependencies could this be an opt-in feature rather than an opt-out feature? (e.g. via `required-features = ['...']`)

Reply:
I've moved it to a separate `example-tests` crate instead to avoid adding a debug-only feature to the root `wasm-bindgen` crate.